authors/apps/usernotifications/handlers.py

```diff
@@ -0,0 +1,94 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.template.loader import render_to_string
+from django.db.models.signals import post_save
+from django.core.mail import send_mail
+from notifications.signals import notify
+from notifications.models import Notification
+from authors.apps.friends.models import Friend
+from authors.apps.authentication.models import User
+from authors.apps.articles.models import Article
+from . import verbs
+
+
+def follow_handler(sender, instance, created, **kwargs):
+    """
+    notification handler for friends
+    """
+    # get the user that has been followed
+    follower = instance.user_from
+    followed = instance.user_to
+    recipient = followed
+    if recipient.email_notification_subscription is False:
+        return
+    url = 'api/profiles/{}/follow/'.format(followed.username)
```

Comment:
The url we send in the notification, shouldn't it be the profile url of the follower?

Reply:
Yes it should. I'll fix that.